rest/route/project_test.go

```diff
@@ -439,6 +440,17 @@ func (s *ProjectPatchByIDSuite) TestRotateAndDeleteProjectPodSecret() {
 	h := s.rm.(*projectIDPatchHandler)
 	h.user = &user.DBUser{Id: "me"}
 
+	smClient, err := cloud.MakeSecretsManagerClient(s.env.Settings())
+
+	s.Require().NoError(err)
+	defer func() {
+		s.Require().NoError(smClient.Close(ctx))
+	}()
+	v, err := cloud.MakeSecretsManagerVault(smClient)
+	s.Require().NoError(err)
+
+	vault := v
```

Comment:
nit: seems like you could just call the above variable `vault` if you're just going to rename it.